config/routes.rb

```diff
@@ -37,8 +37,8 @@
 
     get "/debug" => "debug#index", as: :debug_document
 
-    get "/search-contacts" => "contacts#search", as: :search_contacts
-    post "/search-contacts" => "contacts#insert", as: :insert_contact
+    get "/search-contacts" => "contacts#search"
```

Comment:
Probably something for another PR - it would be good to rename this to just `contacts#index`, as we're not really searching contacts here. Happy to defer, as discussed.

Reply:
Agree, this doesn't reflect the action (which is meant to be insert contact). I'd rather do it as a separate PR if that's fine with you.

Reply:
On a second thought it seems quite couples with these changes, so added two PRs to deal with the route rename and the locales.